lib/utils.js

```diff
@@ -7,6 +7,12 @@ function isPlainObject(value) {
 }
 exports.isPlainObject = isPlainObject;
 
+// Support dynamic require within webpack bundle.
+if (typeof __non_webpack_require__ === 'undefined') {
+  var __non_webpack_require__ = require;
```

Comment:
@benjamn I don't think so. `var` hoisting means only that variable is already set to `undefined` when it's accessed, basically same as 

```javascript
let __non_webpack_require__ = undefined;
if (typeof __non_webpack_require__ === 'undefined') {
  __non_webpack_require__ = require;
}
```

just shorter with ES5 `var`.